package-lock.json

```diff
@@ -524,7 +524,7 @@
     "absolute": {
       "version": "0.0.1",
       "resolved": "https://registry.npmjs.org/absolute/-/absolute-0.0.1.tgz",
-      "integrity": "sha512-Zs3Sz3lmEIH76fvnc4jto4CpuI5bhU2TD+jtBYVIfXnjFm+49y1opKxNy4gtNF+xSLBL7hXRpLwypc+o2MzBIQ==",
+      "integrity": "sha1-wigi+H4ck59XmIdQTZwQnEFzgp0=",
```

Comment:
Looks like various random packages' integrity hashes got downgraded from SHA-512 to SHA-1...

I don't think it'll be a show-stopper, but just wanted to note it here. Fixing it would be kind of a pain ([would need to nuke and re-install all of my NPM dependencies to get them onto the SHA-512 bandwagon](https://stackoverflow.com/a/47834854)).

Reply:
Actually - it looks like the only notable change in package-lock.json was ``datatables.net`` - and its integrity hash is using SHA-512 on my end (makes sense - it's a fresh download).

Literally all the other line diffs are just integrity hash downgrades for pre-existing dependencies.

So I'll go ahead and manually sort it out.